src/js/leaflet.storage.js

```diff
@@ -715,6 +745,41 @@ L.Storage.Map.include({
         L.S.fire('ui:start', {data: {html: container}});
     },
 
+    importRaw: function(rawData, type){
+        var importedData = JSON.parse(rawData);
+
+        for (var option in importedData.properties) {
+            this.options[option] = importedData.properties[option];
```

Comment:
Here we should take care not to change the `storage_id` option value, and add a unittest for that.